src/spikeinterface/core/generate.py

```diff
@@ -799,14 +807,14 @@ def synthesize_random_firings(
         Sampling rate.
     duration : float
         Duration of the segment in seconds.
-    refractory_period_ms: float
+    refractory_period_ms : float
         Refractory period in ms.
-    firing_rates: float or list[float]
+    firing_rates : float or list[float]
         The firing rate of each unit (in Hz).
         If float, all units will have the same firing rate.
-    add_shift_shuffle: bool, default: False
+    add_shift_shuffle : bool, default: False
         Optionally add a small shuffle on half of the spikes to make the autocorrelogram less flat.
-    seed: int, default: None
+    seed : int, default: None
```

Comment:
Reminder (not necessarily now) we need a better docstring for seed. If we do None what does it mean? Right now the beginning user might not understand the point of the seed and what the default seed does.

Reply:
Mmm - good point. We should write one and add it everywhere there's a `seed`.